participants/Filecoin-Crawl.md

```diff
@@ -0,0 +1,63 @@
+# <Filecoin-Crawl>
+
+Before submitting a PR containing your team's project information:
+-Make sure you rename this file to contain your project's name
+-Respond to add your response in each section below
+
+##(0) Project Name and Team Slack Handles
+
+Our project name is Filecoin-Crawl, We'are Filecoin-Crawl.Our team is one of filecoin miner and also we're developing IPFS applications
+
+##(1) Brief Project Description
+
+This is a non-profit organization project that provides available high-quality crawling data for small startups and even individuals. It can help us analyze the world in pursuit of outstanding ideas, which was previously only possible by large search engine companies.
+
+##(2) Link to Project UI
+
+This is our project UI link http://ui.filecoin123.pro:9008   
+This is our project code link https://github.com/IPFS-grandhelmsman/Filecoin-Crawl.git
+
+##(3) What does your application/UI do?
+
+Users can download a variety of high-quality crawling data, users can learn how to use these data, new high-quality crawling data is continuously uploaded for users to use, and users can support the development of the project in the form of donations.
+
+##(4) If your project is using a curated dataset, which one are you using?
+
+The Common Crawl corpus contains petabytes of data collected since 2008. It contains raw web page data, extracted metadata and text extractions.
```

Comment:
This sounds like cool data.  Can you provide more details on this dataset by opening a PR to add it to the official curated dataset repository?

Reply:
The official data set is used，Name：Common Crawl, Descriptions：An open repository of web crawl data, Size：235 TB,Format：WARC, URL:https://commoncrawl.org/